src/main/java/com/craftaro/epicfarming/EpicFarming.java

```diff
@@ -256,6 +256,23 @@ public void onDataLoad() {
 
             this.farmManager.addFarms(this.dataManager.loadBatch(Farm.class, "active_farms"));
             this.boostManager.addBoosts(this.dataManager.loadBatch(BoostData.class, "boosted_players"));
+
+        });
+        Bukkit.getScheduler().runTaskAsynchronously(this, () -> {
+            List<Farm> farms = this.dataManager.loadBatch(Farm.class, "active_farms");
+
+            //System.out.println("Loaded " + farms.size() + " farms from database."); // Debugging
+
+            for (Farm farm : farms) {
+                //System.out.println("Farm ID: " + farm.getId() + " | Location: " + farm.getLocation()); // Debugging
+
+                List<ItemStack> items = DataHelper.loadItemsForFarm(farm.getId());
+                farm.setItems(items);
+
+                //System.out.println("Farm ID " + farm.getId() + " has " + items.size() + " items."); // Debugging
+            }
```

Comment:
Personally not a big fan of commented-out code, especially if they mostly look temporary.

I would prefer seeing them removed or if desired, we can introduce a setting in the config to enable debug logging and have some Debug-Logging-Class in the Core to help with that.